my-app/src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+ const Calculator =  () => {
+  
+  const numbers = [7, 8, 9, 4, 5, 6, 1, 2, 3, 0];
+  const operators = ["÷", "*", "-", "+"];
```

Comment:
- [ ] It would be better if the Calculator.js component is class-based, as this is one of the requirements for this milestone. Kindly fix this minor issue.

Reply:
- [ ] Kindly make the Pull Request description more descriptive.

Reply:
- [ ] Kindly add `npm install` to the setup section of the README file and also add a good screenshot.